invite/forms.py

```diff
@@ -115,6 +117,19 @@ def clean(self, *args, **kwargs):
         self.clean_email()
         return self.cleaned_data
 
+class BaseInviteItemFormSet(forms.BaseFormSet):
+    def clean(self):
+        """
+        Validates that there are no duplicate email addresses across forms.
+        """
```

Comment:
```suggestion
        """Validate there are no duplicate email addresses across forms."""
```

Reply:
I was also suggesting that this should [all be on one line](https://peps.python.org/pep-0257/#one-line-docstrings).